crates/topos-tce-broadcast/src/lib.rs

```diff
@@ -251,6 +161,10 @@ impl ReliableBroadcastClient {
     ) -> Result<(), ()> {
         let broadcast_commands = self.broadcast_commands.clone();
 
+        if broadcast_commands.capacity() <= *constant::COMMAND_CHANNEL_CAPACITY {
```

Comment:
Can we comment this code? To easily understand why we increase the channel capacity here and when this will be the case.

Reply:
Here we just increase the metrics, not the channel

Reply:
Ha, exactly why a comment might be helpful I think :)